### PR TITLE
Optional override of runner image in chart

### PR DESCRIPTION
**Pull request by sledigabel**
This commit adds the option to override the actions runner image. This
allows running the controller in environments where access to Dockerhub
is restricted.

It uses the parameter [--runner-image](https://github.com/actions-runner-controller/actions-runner-controller/blob/master/main.go#L89) from the controller.
The default value is set as a constant
[here](https://github.com/actions-runner-controller/actions-runner-controller/blob/acb906164b6cc8b3604fa816cd3a99acffe991a2/main.go#L40).

The default value for the chart is the same.

### CODE REVIEW

File: charts/actions-runner-controller/templates/deployment.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ spec:
         - "--enable-leader-election"
         - "--sync-period={{ .Values.syncPeriod }}"
         - "--docker-image={{ .Values.image.dindSidecarRepositoryAndTag }}"
+        - "--runner-image={{ .Values.image.actionsRunner }}"
```
Comment (mumoshu) [inline charts/actions-runner-controller/templates/deployment.yaml:43]:
```suggestion
        - "--runner-image={{ .Values.image.actionsRunnerRepositoryAndTag }}"
```

File: charts/actions-runner-controller/values.yaml [hunk 2]
```diff
@@ -29,7 +29,8 @@ authSecret:
   #github_token: ""
 
 image:
-  repository: actions-runner-controller/actions-runner-controller
+  repository: "actions-runner-controller/actions-runner-controller"
+  actionsRunner: "summerwind/actions-runner:latest"
```
Comment (mumoshu) [inline charts/actions-runner-controller/values.yaml:33]:
```suggestion
  actionsRunnerRepositoryAndTag: "summerwind/actions-runner:latest"
```

File: charts/actions-runner-controller/README.md [hunk 3]
```diff
@@ -24,6 +24,7 @@ _Default values are the defaults set in the charts values.yaml, some properties
 | `authSecret.github_token`                                | Your chosen GitHub PAT token. **This can't be set at the same time as the `authSecret.github_app_*`**                      |                                                                      |
 | `image.repository`                                       | The "repository/image" of the controller container                                                                         | actions-runner-controller/actions-runner-controller                  |
 | `image.tag`                                              | The tag of the controller container                                                                                        |                                                                      |
+| `image.actionsRunner`                                    | The "repository/image" of the actions runner container                                                                     | summerwind/actions-runner:latest                                     |
```
Comment (mumoshu) [inline charts/actions-runner-controller/README.md:27]:
```suggestion
| `image.actionsRunnerRepositoryAndTag `                   | The "repository/image" of the actions runner container                                                                     | summerwind/actions-runner:latest                                     |
```

**Review by mumoshu — CHANGES_REQUESTED** [review]
@sledigabel Hey! Thanks a lot for your contribution. This looks almost good 👍 The only thing I wanted was renaming `actionsRunner` to `actionsRunnerRepositoryAndTag` for consistency with the existing `dindSidecarRepositoryAndTag`. Would you mind confirming my proposed changes?

**Comment by sledigabel** [comment]
All updated, thanks for your review @mumoshu !

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks a lot for the addition @sledigabel ☺️

**Pull request merged**